Taskfile.yml

```diff
@@ -147,6 +147,7 @@ tasks:
     deps:
       - test-integration-pg
       - test-integration-tigris
+      - test-integration-cockroach
```

Comment:
Let's not run them by default, but please update task's description ("all" is not valid in that case)

Reply:
"Run all stable integration tests"?
"Run integration tests for PostgreSQL and Tigris"?

Reply:
I would mention names, yeah